### DISCUSSION TITLE
Log#5 | 삽질을 존중하는 문화 | kciter.so

### DISCUSSION THREAD

**Discussion by giscus** [kciter/kciter.github.io#32] [Announcements]
# Log#5 | 삽질을 존중하는 문화 | kciter.so

개발자들이 삽질한다는 얘기를 들어보셨나요? 실제로 보면 열심히 뭔가를 하는데 왜 ‘삽질’이라고 부를까요? 제가 느끼기에 개발자는 전반적으로 자조적인 성향이 짙습니다. 아마 출시한 제품에 대한 책임감 때문이 아닐까 싶습니다. 기능에 문제가 생기면 결국…

https://kciter.so/posts/log-5/

**Comment by ibcylon** [comment]
⛏️
